synapse/cores/sqlite.py

```diff
@@ -143,6 +143,7 @@ class SqliteStorage(s_cores_storage.Storage):
     _t_getrows_by_iden_prop_strval = 'SELECT * FROM {{TABLE}} WHERE iden={{IDEN}} AND prop={{PROP}} AND strval={{VALU}}'
 
     _t_getrows_by_iden_range = 'SELECT * FROM {{TABLE}} WHERE iden >= {{LOWERBOUND}} and iden < {{UPPERBOUND}}'
+    _t_getrows = 'SELECT * FROM {{TABLE}} LIMIT -1 OFFSET {{OFFSET}}'
```

Comment:
We should make sure that this query isn't affected by the SQLITE LIMIT weirdness listed here: https://github.com/vertexproject/synapse/issues/167